### PR TITLE
Fix for =< 5.0 native firms

### PR DESCRIPTION
**Pull request by Pirater12**
On pre 5.0 firms soundhax doesn't work as the va to pa translation changed. By modifying the code_va_to_pa function we(@saibotu and I) were able to make it work again(confirmed to work on 3.0, 4.1 and 4.5). The soundhax bug seems to exist on 2.X soundapp too but the rop gadgets and va_to_pa func need to modified to make it work. Special thanks to @jason0597, @ihaveamac @saibotu for testing it out.
The changes I made to the site can be viewed here:- https://pirater12.github.io/index.html

### CODE REVIEW

**Comment by nedwill** [comment]
This looks great! @yellows8 added you as a reviewer - do you have any device with old firm to test on quickly, or can you confirm those offsets? I can't review right away but I should have time next week.

**Comment by Pirater12** [comment]
@nedwill You are talking about (4.X-5.X) offsets, right? They have already been confirmed to be working.

File: exp.py [hunk 1]
```diff
@@ -8,13 +8,16 @@
 
 REGION = "usa"
 TYPE = "old" # "new"
-
+FIRM = "pre5" # "post5"
```
Comment (yellows8) [inline exp.py]: Default to post5?

Comment (Pirater12) [reply exp.py]: fixed!

**Comment by yellows8** [comment]
@nedwill Maybe with nandredir but not interested in testing.

**Comment by nedwill** [comment]
Meh, same here. I'll just merge since the submitter claims to have tested it with a few people and this looks pretty high quality. Thanks again!

**Comment by jason0597** [comment]
Yes I've helped in testing 
So has @saibotu

**Pull request merged**